pkg/localstore/reserve.go

```diff
@@ -164,3 +164,40 @@ func withinRadius(db *DB, item shed.Item) bool {
 	po := db.po(swarm.NewAddress(item.Address))
 	return po >= item.Radius
 }
+
+// ComputeReserveSize iterates on the pull index to count all chunks
+// starting at some proximity order with an generated address whose PO
+// is used as a starting prefix by the index.
+func (db *DB) ComputeReserveSize(startPO uint8) (uint64, error) {
+
+	var count uint64
+
+	err := db.pullIndex.Iterate(func(item shed.Item) (stop bool, err error) {
+		count++
+		return false, nil
+	}, &shed.IterateOptions{
+		StartFrom: &shed.Item{
+			Address: generateAddressAt(db.baseKey, int(startPO)),
+		},
+	})
+
+	return count, err
+}
+
+func generateAddressAt(baseBytes []byte, prox int) []byte {
```

Comment:
This function definition seems to be more suitable to be in `swarm` package, together with other `swarm.Address` utilities. 

Also, use of `swarm.Address` type instead of raw `[]byte` would be more indicative.